distributed/protocol/tests/test_torch.py

```diff
@@ -31,3 +31,10 @@ def test_resnet():
     header, frames = serialize(model)
     model2 = deserialize(header, frames)
     assert str(model) == str(model2)
+
+
+def test_deserialize_grad():
+    torch.set_printoptions(precision=10)
```

Comment:
My guess is that this line was used for debugging.  Maybe we can remove it before we merge?